homeassistant/components/shelly/entity.py

```diff
@@ -525,16 +499,6 @@ def attribute_value(self) -> StateType:
             )
         return self._last_value
 
-    @property
-    def extra_state_attributes(self) -> dict[str, Any] | None:
-        """Return the state attributes."""
-        if self.entity_description.extra_state_attributes is None:
-            return None
-
-        return self.entity_description.extra_state_attributes(
-            self.block_coordinator.device.status
-        )
-
```

Comment:
Currently we don't have any REST (Gen1) sensors which uses `extra_state_attributes` and since we try to minimize the use of them in core it is better to remove this code